website/docs/topics/llm_configuration.ipynb

```diff
@@ -397,6 +293,112 @@
     "}"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using Azure Active Directory (AAD) Authentication\n",
+    "\n",
+    "Azure Active Directory (AAD) provides secure access to resources and applications. Follow the steps below to configure AAD authentication for Autogen.\n",
+    "\n",
+    "#### Prerequisites\n",
+    "- An Azure account with AAD configured.\n",
+    "- Appropriate permissions to register an application in AAD.\n",
+    "\n",
+    "#### Step 1: Register an Application in AAD\n",
+    "1. Navigate to the [Azure portal](https://portal.azure.com/).\n",
+    "2. Go to `Azure Active Directory` > `App registrations`.\n",
+    "3. Click on `New registration`.\n",
+    "4. Enter a name for your application.\n",
+    "5. Set the `Redirect URI` (optional).\n",
+    "6. Click `Register`.\n",
+    "\n",
+    "#### Step 2: Configure API Permissions\n",
+    "1. After registration, go to `API permissions`.\n",
+    "2. Click `Add a permission`.\n",
+    "3. Select `Microsoft Graph` and then `Delegated permissions`.\n",
+    "4. Add the necessary permissions (e.g., `User.Read`).\n",
+    "\n",
+    "#### Step 3: Obtain Client ID and Tenant ID\n",
+    "1. Go to `Overview` of your registered application.\n",
+    "2. Note down the `Application (client) ID` and `Directory (tenant) ID`.\n",
+    "\n",
+    "Note: For the first 3 steps, For detailed and up-to-date instructions, please refer to the official [Azure OpenAI documentation](https://learn.microsoft.com/en-us/azure/ai-services/openai/).\n",
```

Comment:
Thank you very much for the updates, @prithvi2226 . Could you share the link to the docs of the steps you described here?
I'm asking because I don't see docs like step 1 to step 3 in `https://learn.microsoft.com/en-us/azure/ai-services/openai/`. It looks like  the authenticate doc is https://learn.microsoft.com/en-us/azure/api-management/api-management-authenticate-authorize-azure-openai and the doc for Microsoft Entra ID (the new name of AAD) authentication is https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/managed-identity

Did I miss anything? Thanks.

Reply:
I am sorry! You are right, it was a different link I posted. I was careless. I will take the necessary steps to edit and change it. Thanks again for your guidance!

Reply:
Hello, will this be released soon?

Reply:
This is blocked by the change request from @qingyun-wu . @qingyun-wu , could you please take a look again?